### PR TITLE
Sztab-WOSP-Bialystok/wosp-staff-issues#11

### PR DESCRIPTION
**Pull request by TheUnDc**

### CODE REVIEW

File: app/concerts/page.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+						</p>
+					</Col>
+				</Row>
+				<Row>
+					<Col xl={{ span: 8, offset: 2 }} className="my-3">
+						<Current></Current>
+					</Col>
+				</Row>
+			</Container>
+		</Row>
```
Comment (Freesebee) [inline app/concerts/page.tsx:31]:
```suggestion
```

File: app/concerts/concerts/current/current-data.json
Comment (Freesebee) [inline app/concerts/concerts/current/current-data.json:1]: File can be deleted - concerts poster will be used

File: app/concerts/concerts/current/current.tsx
Comment (Freesebee) [inline app/concerts/concerts/current/current.tsx:1]: File can be deleted - concerts poster will be used

File: app/concerts/page.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+							elit. Omnis veritatis est quidem quo dignissimos
+							nisi enim commodi ducimus cupiditate totam illo
+							voluptatibus voluptatum quas id soluta impedit,
+							ratione inventore esse.
+						</p>
+					</Col>
+				</Row>
+				<Row>
+					<Col xl={{ span: 8, offset: 2 }} className="my-3">
+						<Current></Current>
```
Comment (Freesebee) [inline app/concerts/page.tsx:26]: I added concerts poster image in branch [11-concerts-page](https://github.com/Sztab-WOSP-Bialystok/wosp-staff/tree/11-concerts-page). Scale it down to match page main Container component and upload like in [Honorary Patronage](https://github.com/Sztab-WOSP-Bialystok/wosp-staff/blob/11-concerts-page/app/home/sections/honorary-patriage/honorary-patronage-section.tsx)
```suggestion
						<img src={concertsPoster.src} width={"100%"} />
```

**Review by Freesebee — CHANGES_REQUESTED** [review]
🏅 🔥

**Review by Kompanel — APPROVED** [review]

File: app/concerts/concerts/current/current-data.json
Comment (TheUnDc) [reply app/concerts/concerts/current/current-data.json:1]: done

File: app/concerts/concerts/current/current.tsx
Comment (TheUnDc) [reply app/concerts/concerts/current/current.tsx:1]: done

**Review by Freesebee — APPROVED** [review]
❤️

**Pull request merged**